python/assistant-stream/src/assistant_stream/modules/tool_call.py

```
@@ -39,22 +38,34 @@ def append_args_text(self, args_text_delta: str) -> None:
+        Deprecated: Use set_response() instead.
+        """
+        import warnings
+        warnings.warn(
+            "set_result() is deprecated. Use set_response() instead.",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        return self.set_response(result)
+
+    def set_response(
+        self, result: Any, *, artifact: Any | None = None, is_error: bool = False
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `result`

(ANN401)